packages/select/src/Select/components/Trigger/SingleTriggerButton/SingleTriggerButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import classnames from "classnames"
+import { RootProps } from "../../Root"
+import { TriggerButtonBase } from "../TriggerButtonBase"
+import styles from "./SingleTriggerButton.module.scss"
+
+export type SingleTriggerButtonProps = {
+  placeholder: string
+  selectedOptionLabel: RootProps["selectedKey"]
+  classNameOverride?: string // TODO: migrate it to use OverrideClassName<T> and omit the props controlled by React-Aria
+}
+
+export const SingleTriggerButton: React.VFC<SingleTriggerButtonProps> = ({
```

Comment:
I understand this a pattern copied from FilterMultiSelect which a bit more complicated that just the span with text, you could move this logic to TriggerButtonBase and have one less component to maintain